xdr/next/Stellar-contract.x

```diff
@@ -168,17 +168,17 @@ case SST_OK:
 case SST_UNKNOWN_ERROR:
     SCUnknownErrorCode unknownCode;
 case SST_HOST_VALUE_ERROR:
-    SCHostValErrorCode errorCode;
+    SCHostValErrorCode valCode;
 case SST_HOST_OBJECT_ERROR:
-    SCHostObjErrorCode errorCode;
+    SCHostObjErrorCode objCode;
 case SST_HOST_FUNCTION_ERROR:
-    SCHostFnErrorCode errorCode;
+    SCHostFnErrorCode fnCode;
 case SST_HOST_STORAGE_ERROR:
-    SCHostStorageErrorCode errorCode;
+    SCHostStorageErrorCode storageCode;
 case SST_HOST_CONTEXT_ERROR:
-    SCHostContextErrorCode errorCode;
+    SCHostContextErrorCode contextCode;
 case SST_VM_ERROR:
-    SCVmErrorCode errorCode;
+    SCVmErrorCode vmCode;
```

Comment:
Did you manually copy these changes from `stellar-xdr-next` or were these picked up automatically?
I guess it's not clear to me is, why do we have these local files if the source of changes should come from `stellar-xdr-next`? Wouldn't it make sense for them to live in the same repo/workspace? 
When I makes changes to the `.x` files in the `stellar-xdr-next`, it would be nice for me to run the regen code to make sure no syntactical errors, which makes it feel more natural to make those changes here.